readme/cdh-s3.md

```diff
@@ -0,0 +1,54 @@
+# WANdisco Fusion
+
+## Overview
+
+This repository contains a docker-compose configuration that can be used to deploy a fully functional version of WANdisco Fusion with a 30 day evaluation license:
+
+- CDH Sandbox to S3
+
+Full documentation for this case can be found in our [Quickstart Guide](https://wandisco.github.io/wandisco-documentation/docs/quickstarts/installation/cdh_sandbox-s3_ld)
```

Comment:
This link will be changing soon once the standalone OneUI + Fusion 2.15.5 releases are live on master:

https://wandisco.github.io/wandisco-documentation/docs/quickstarts/installation/cdh_sandbox-s3

Reply:
Updated